doc/poo.md

```diff
@@ -343,13 +343,23 @@ In that list:
     You can specify the empty list `[]` if there are no such super-prototypes.
 
   * The optional following list of parameters are as many of slot names that
-    will be bound to macros to access the relevant slots of the instance being
```

Comment:
Nah, it's bound *to* macros, i.e. the identifier will not refer to a variable, but to syntax that expands to some expression that implements access to the slot.